Specs/Data/Models/moving-box/moving-box.gltf

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
What is unique about this new test model?

I'm fine with adding it, I just want to know what it is used for...and if we should include it in the glTF repo.